### ISSUE TITLE
Why 1-mask? Error code?

### ISSUE THREAD

**Issue by KyujinHan** [alimama-creative/FLUX-Controlnet-Inpainting#46]
Hello!  
Thanks for your inpainting model! It is pretty good!

I wonder that why applied `1-mask`?   
[ControlNet, prepare_image_with mask function](https://github.com/alimama-creative/FLUX-Controlnet-Inpainting/blob/6ea5d92d82c47cc184f4eb545534226c92cb23da/pipeline_flux_controlnet_inpaint.py#L685C9-L688C24)

```python
mask = torch.nn.functional.interpolate(
            mask, size=(height // self.vae_scale_factor * 2, width // self.vae_scale_factor * 2)
        )
mask = 1 - mask  # why????

control_image = torch.cat([image_latents, mask], dim=1)

```
  
I understand that, the original input of mask has 0 or 1. (Here 1 means: want to change object)  
But following above code, `1-mask` is applied.  
In this case, I think that white region is background and black region is object.  

So, it has opposite meaning of masking.  
Could you explain more about the `1-mask` meaning??
  
Thank you!
